release_notes/migrating_to_0.20.md

```diff
@@ -0,0 +1,41 @@
+# NGS CLI v 0.20.0
+
+Version 0.20 brings compatibility with JWT v2 functionality added to the NATS ecosystem. Future updates to NGS will introduce features like JetStream which will require version 2 if not deployed as a leaf node.
+
+NGS is fully compatible with JWT v1 or v2 projects. However, the tooling that is used to manage your account (`ngs` cli) and your configurations (`nsc`) determines the version `ngs` cli that you can use, so you should update now, simply to have it at the current version.
+
+However, by default the `ngs` cli installer will default to `nsc` at it's latest version. If you are starting a new project, stop reading as the tooling will be fully compatible.
+
+If you are using `nsc` 0.5.0, you'll want to ensure that the installer didn't install a newer version of `nsc`. You can easily downgrade by executing the command: `nsc update --version 0.5.0`.
+
+If you are using `nsc` 2.x but tried `ngs` in the past, you'll want to ensure you update your `ngs` tool to its current version.
+
+Finally, note that if you have on-prem servers, only `nats-server` 2.2.0 or better support v2. If you are running older servers, don't upgrade your configurations, and stay on nsc 0.5.0.
+
+
+
+The following matrix describes compatibility between `nsc`, `ngs cli`:
+
+|              | JWT v1               | JWT v2             |
+|---           |:---:                 |---                 |
+| nsc 0.5.0    |  :heavy_check_mark:  | :x:                |
+| nsc 2.2.3    |  :x:                 | :heavy_check_mark: |
+| ngs cli 0.12 |  :heavy_check_mark:  | :x:                |
+| ngs cli 0.20 |  :heavy_check_mark:  | :heavy_check_mark: |
+
+
+If you see an error like `Error: error reading root: unexpected "ed25519-nkey" algorithm` coming from `nsc` or `ngs` cli, you have a mismatch. The JWTs you are trying to read are version 2. 
+
+
+
+# Upgrading your project to use v2 and nsc 2.x
+
+First back up your store and keys, run `nsc env` and note where the `Stores Dir` and `$NKEYS_PATH` are looking for data. 
```

Comment:
It might be good to have a text block of the commands and output of doing this from a prompt with the defaults to connect all of the dots:  e.g. using nsc to view the env, cp -R the dir, nsc update, then nsc add operator -f...